presto-verifier/src/main/java/com/facebook/presto/verifier/checksum/ChecksumResult.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.verifier.checksum;
+
+import com.facebook.presto.spi.type.SqlVarbinary;
+import com.google.common.annotations.VisibleForTesting;
+
+import java.sql.ResultSet;
+import java.sql.ResultSetMetaData;
+import java.sql.SQLException;
+import java.sql.Types;
+import java.util.HashMap;
+import java.util.Map;
+
+import static com.google.common.base.Preconditions.checkArgument;
+import static com.google.common.base.Preconditions.checkState;
+import static java.util.Collections.unmodifiableMap;
+import static java.util.Objects.requireNonNull;
+
+public class ChecksumResult
```

Comment:
We aren't just running checksums. How about naming this as `QueryValidationOutput`. Since in this case, we executed a validation query (which consists of checksums and more)

Reply:
Same as above, addressed here: https://github.com/prestodb/presto/pull/12489#issuecomment-475096230